sql/synth_recipes.sql

```diff
@@ -1060,7 +1060,7 @@ INSERT INTO `synth_recipes` VALUES (14032,0,0,31,88,0,0,0,0,0,0,4096,4238,654,65
 INSERT INTO `synth_recipes` VALUES (14033,0,0,0,89,0,0,0,0,0,0,4096,4238,2000,0,0,0,0,0,0,0,2001,2001,2001,2001,1,1,1,1,'Drk. Adm. Sheet');
 INSERT INTO `synth_recipes` VALUES (14034,0,0,0,89,0,0,0,0,0,0,4098,4240,2000,0,0,0,0,0,0,0,3306,3306,3306,3306,6,8,10,12,'Dark Adaman Bolt Heads');
 INSERT INTO `synth_recipes` VALUES (14035,0,1995,0,89,0,0,0,0,0,0,4096,4238,2000,2000,2000,2000,2000,2000,2144,0,2001,2001,2001,2001,6,6,6,6,'Drk. Adm. Sheet');
--- INSERT INTO `synth_recipes` VALUES (14036,0,0,255,89,54,0,0,0,0,0,4096,4238,650,654,654,711,745,0,0,0,17251,17264,17264,17264,1,1,1,1,'Hellfire');
+INSERT INTO `synth_recipes` VALUES (14036,0,0,34,89,54,0,0,0,0,0,4096,4238,650,654,654,711,745,0,0,0,17251,17264,17264,17264,1,1,1,1,'Hellfire');
```

Comment:
https://ffxiclopedia.fandom.com/wiki/Hellfire
https://www.bg-wiki.com/ffxi/Hellfire
https://www.ffxiah.com/item/17251/hellfire

No source agrees on WW lvl. It's probably better to use he highest lvl. In this case, 41.